tidb-configuration-file.md

```diff
@@ -636,9 +636,4 @@ TiDB 服务状态相关配置。
 
 ## experimental
```

Comment:
Can we delete the ‘experimental' part? I don't think it will be used in the future.